### ISSUE TITLE
Add another dataset other than Imdb

### ISSUE THREAD

**Issue by ldneal** [Recode-Hive/Scrape-ML#41]
Sir we need to change the database of all programming scrape in Python. SO Pls Assign me in this project.

**Comment by sanjay-kv** [comment]
Can you be spesfic on this part? What changes, and how it will be efficient than the current one. with ss. Thanks

**Comment by ldneal** [comment]
Yes Sir I will solve this code error with minor changes of program solution PLS assign me
